### ISSUE TITLE
[Installation Status Updater] Valorant and Legends of Runeterra keeps being maked as installed but they are not installed

### ISSUE THREAD

**Issue by LordBoos** [darklinkpower/PlayniteExtensionsCollection#522]
### Extension name

Installation Status Updater

### Bug Description

Valorant and Legends of Runeterra keeps being maked as installed but they are not installed. This happens on every library update.

### To Reproduce

1. Have Riot client installed
2. Add Riot library integration
3. Update Library

### Logs

[extensions.log](https://github.com/darklinkpower/PlayniteExtensionsCollection/files/14791670/extensions.log)
[playnite.log](https://github.com/darklinkpower/PlayniteExtensionsCollection/files/14791673/playnite.log)

### Screenshots

![image](https://github.com/darklinkpower/PlayniteExtensionsCollection/assets/939813/6961ad3d-3237-4187-9666-11deb3d7b968)
![image](https://github.com/darklinkpower/PlayniteExtensionsCollection/assets/939813/fde7b33e-fa1c-49e0-935d-024fbd7d1f56)

**Comment by darklinkpower** [comment]
Please share screenshots of the games Installation and Actions tabs.

**Comment by LordBoos** [comment]
I tried disabling this option, but the issue persist.
![image](https://private-user-images.githubusercontent.com/1389286/317927143-029e1c98-eabb-4495-a8be-cf17e8994a90.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE2ODA0NjcsIm5iZiI6MTcxMTY4MDE2NywicGF0aCI6Ii8xMzg5Mjg2LzMxNzkyNzE0My0wMjllMWM5OC1lYWJiLTQ0OTUtYThiZS1jZjE3ZTg5OTRhOTAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMyOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMjlUMDI0MjQ3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YzVkZTExZGJiMWM4NGJmZjhhZGNkN2VmNTM0ZjZjNWYzMmY3OThkMWViNzUxZWYwYzJlMzM5NGU3MDgwMDYzMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.JYN-cOaAaAqUVIeH8jA9CWGm3i8e7RJ0JTVmP2wlf9g)

![image](https://github.com/darklinkpower/PlayniteExtensionsCollection/assets/939813/8fdd4a6a-cdf8-4f58-8b89-61a7694da3ec)
![image](https://github.com/darklinkpower/PlayniteExtensionsCollection/assets/939813/120b6635-8f4c-400d-9af4-83348b36a417)

**Comment by darklinkpower** [comment]
I don't know that language but if "Soubor" means `File`, then the reason they are being marked as installed is because the Play Action points to that `[...]RiotClientServices.exe` file and if it exists in your system then they will be marked as installed. If you need to have that Action there you can unmark the game as installed and afterwards exclude them from the extension here in the right click menu:

![image](https://github.com/darklinkpower/PlayniteExtensionsCollection/assets/1389286/e4621745-d009-4a52-958b-e8e9fa00c287)

**Comment by LordBoos** [comment]
Yes, Soubor means File. Thing is that Play Action pointing to [...]RiotClientServices.exe is added automatically by Riot library integration. Adding those games to ignore list helped.
